ext/node/ops/sqlite/database.rs

```diff
@@ -177,6 +334,34 @@ fn open_db(
   Ok(conn)
 }
 
+fn database_constructor(
+  _: &mut v8::HandleScope,
+  args: &v8::FunctionCallbackArguments,
+) -> Result<(), validators::Error> {
+  // TODO(littledivy): use `IsConstructCall()`
```

Comment:
Do we have a rusty_v8 issue for that?

Reply:
https://github.com/denoland/rusty_v8/issues/1791